### PR TITLE
Format the docstring with better indentation

### PR DESCRIPTION
**Pull request by wonderfly**
@cmccoy @ehankland I am not sure what style guide this repository is following, but the docstring at the entrance file looks like it could be improved.

### CODE REVIEW

File: perfkitbenchmarker/pkb.py [hunk 1]
```diff
@@ -30,27 +30,24 @@
...
+
 prepare: PerfKitBenchmarker will read command-line flags, decide which
-benchmarks to run
-         and create necessary resources for each benchmark, including networks,
-         VMs, disks, keys and execute the Prepare function of each benchmark to
-         install necessary softwares, upload datafiles, etc and generate a
-         run_uri, which can be used to run benchmark multiple times.
+         benchmarks to run and create necessary resources for each benchmark,
+         including networks, VMs, disks, keys and execute the Prepare function
+         of each benchmark to install necessary softwares, upload datafiles, etc
```
Comment (voellm) [inline perfkitbenchmarker/pkb.py]: softwares -> software

**Comment by voellm** [comment]
LGTM after "softwares" fix.

**Comment by voellm** [comment]
Thanks for the edits!

File: perfkitbenchmarker/pkb.py (hunk 1 above)
Comment (wonderfly) [inline perfkitbenchmarker/pkb.py]: Done

**Comment by wonderfly** [comment]
Addressed the "softwares" typo. PTAL.

**Pull request merged**